parent/pom.xml

```diff
@@ -726,7 +726,6 @@
                                 <requireJavaVersion>
                                     <version>[${minimum.java.version},)</version>
                                 </requireJavaVersion>
-                                <DependencyConvergence />
```

Comment:
room has some libraries with kotlin 1.8.0 and some with kotlin 1.9.0, causing conflicts. Maybe someone who knows maven better than me knows how to resolve that conflict, but just to keep going I turned it off so I can first focus on the rest